src/models/user-token-profile.model.ts

```diff
@@ -1,9 +1,20 @@
 import { UserRole } from './user.db.model';
 
+export interface JWTTokenPayload {
+    id: string;
+    username: string;
+    role: UserRole;
+    firstname: string;
+    lastname: string;
+    exp: number;
+    class_id?: string;
+}
+
 export interface UserTokenProfile {
     id: string;
     username: string;
     role: UserRole;
     firstname: string;
     lastname: string;
+    class_id?: string;
 }
```

Comment:
JWTTokenPayload and UserTokenProfile are almost the same except exp and lastname? when we need to use each one of them? there isn't any comment of explanation